package.json

```diff
@@ -7,10 +7,10 @@
   "bin": {
```

Comment:
Bump package version

Reply:
Run the `prepublish` command before commiting for auto generating some of the required files

Reply:
added to do